site/content/docs/5.0/examples/pricing/pricing.rtl.css

```diff
@@ -0,0 +1,37 @@
+@font-face {
+  font-family: Dubai;
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url("https://www.fontstatic.com/fonts/dubai-light/dubai-light.eot?#iefix") format("eot"), url("https://www.fontstatic.com/fonts/dubai-light/dubai-light.woff") format("woff");
+}
+
+@font-face {
+  font-family: Dubai;
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("https://www.fontstatic.com/fonts/dubai/dubai.eot?#iefix") format("eot"), url("https://www.fontstatic.com/fonts/dubai/dubai.woff") format("woff");
+}
```

Comment:
We usually try to avoid using custom fonts: doesn't our system font stack fit our needs?
Should we consider using another font stack for RTL?

Reply:
In fact  we used one in our Blog example, however we only make the Google Fonts call: there's no need to provide a`@font-face` declaration.

Reply:
I was going to bring up this point. The current font stack is not good with Arabic. We would need to improve fonts and typography for RTL. However, each language would have its own fonts, so the task would be a bit more involved.

Reply:
I was going to use a single import statement for the fonts, however, even though they're the same family, their `font-family` where different (i.e. Dubai, Dubai-light, Dubai-heavy ...), which would make it unintuitive when using `<strong>` and `<small>` or the typed font sizes. I contacted the guy who maintains the CDN (https://www.fontface.me/) to fix this issue and he agreed to do it but I'm still waiting for him to publish the changes.

Reply:
Take a look at https://www.google.com/get/noto/

Reply:
Based on your first comment, I'd prefer we tackle our core font-stack improvements if possible, and at least provide some guidance in our docs, before using custom fonts in our examples (which should not require it, IMHO).

Is there some system / websafe font stack that would be commonly used—for Arabic at least?

Reply:
Did you get a chance to look at https://www.google.com/get/noto/ ?